time_tracker_api/projects/projects_namespace.py

```diff
@@ -58,53 +69,42 @@ def get(self):
         return project_dao.get_all(), 200
 
     @ns.doc('create_project')
+    @ns.response(409, 'This project already exists')
+    @ns.response(422, 'The data has an invalid format')
     @ns.expect(project_input)
     @ns.marshal_with(project, code=201)
     def post(self):
         """Create a project"""
         return project_dao.create(ns.payload), 201
 
+
 # TODO : fix, this parser is for a field that is not being used.
 project_update_parser = ns.parser()
-project_update_parser.add_argument('active',
-                                   type=inputs.boolean,
-                                   location='form',
-                                   required=True,
-                                   help='Is the project active?')
 
 
 @ns.route('/<string:id>')
 @ns.response(404, 'Project not found')
 @ns.param('id', 'The project identifier')
 class Project(Resource):
     @ns.doc('get_project')
+    @ns.response(422, 'The id has an invalid format')
     @ns.marshal_with(project)
     def get(self, id):
         """Retrieve a project"""
         return project_dao.get(id)
 
-    @ns.doc('update_project_status')
-    @ns.expect(project)
-    @ns.response(204, 'State of the project successfully updated')
-    def post(self, id):
-        """Updates a project using form data"""
-        try:
-            update_data = project_update_parser.parse_args()
-            return project_dao.update(id, update_data), 200
-        except ValueError:
-            abort(code=400)
-        except MissingResource as e:
-            abort(message=str(e), code=404)
-
-    @ns.doc('put_project')
+    @ns.doc('update_project')
+    @ns.response(422, 'The data has an invalid format.')
+    @ns.response(409, 'This project already exists')
     @ns.expect(project_input)
     @ns.marshal_with(project)
     def put(self, id):
-        """Create or replace a project"""
+        """Updates a project"""
```

Comment:
No comment needed here or is this going to be used by swagger?

Reply:
Yes, I found these comments here are used by swagger.

Reply:
cool